skyvern/__init__.py

```
@@ -46,6 +46,7 @@ def process_trace(self, trace: list[Span]) -> list[Span] | None:
     run_task,  # noqa: E402
     send_email,  # noqa: E402
     upload_file,  # noqa: E402
+    validate,  # noqa: E402
```

Ruff (0.13.2)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)